hs-bindgen/hs-bindgen.cabal

```diff
@@ -250,12 +251,23 @@ executable clang-ast-dump
       -- External dependencies
     , optparse-applicative >= 0.18 && < 0.19
 
-test-suite golden
+test-suite tests
```

Comment:
`tests` is a terrible name, partly because it's a magic keyword in `cabal-install` selector syntax.

Use something more specific, `hs-bindgen-tests`.